src/seg_utils.py

```diff
@@ -333,21 +333,27 @@ def train_segmentation(
     return net, results
 
 # Segmentation
-def segment(net, device, inference_loader, qlty_object):
+def segment(net, device, inference_loader, qlty_object, tiled_client):
     net.to(device)   # send network to GPU
-    patch_preds = [] # store results for patches
+    frame_number = 0
     for idx, batch in enumerate(inference_loader):
-        print(f'{idx+1}/{len(inference_loader)}')
+        print(f'Batch: {idx+1}/{len(inference_loader)}')
         with torch.no_grad():
             # Necessary data recasting
             batch = batch.type(torch.FloatTensor)
             batch = batch.to(device)
             # Input passed through networks here
-            output_network = net(batch)
-            patch_preds.append(output_network)
-    
-    patch_preds = torch.concat(patch_preds)
-    stitched_result, weights = qlty_object.stitch(patch_preds)
-    # Individual output passed through argmax to get predictions
-    seg = torch.argmax(stitched_result.cpu(), dim=1).numpy()
-    return seg
\ No newline at end of file
+            patch_output = net(batch)
+            stitched_result, weights = qlty_object.stitch(patch_output)
```

Comment:
I got an error here since `stitched_result` is in CPU while the `patch_output` is still in cuda. To avoid this, you could send `patch_output = patch_output.cpu()` before stitching

Reply:
Addressed